_posts/entities/2022-05-10-func_slide.md

```diff
@@ -0,0 +1,25 @@
+---
+title: func_slide
+category: entity
+tags:
+ - slide
+ - func
+---
+
+----
+A brush entity that allows the player to slide across the surface of it without friction. 
+
+## Keyvalues
+
+>**Keep player on slide surface** (StayOnSlide&lt;**boolean**&gt;)
+
+ Forces the player to stay on the brush, even if it suddenly curves downwards or launches the player upwards.
```

Comment:
I would use the descriptions from the FGD to keep them in sync: https://github.com/ChaosInitiative/Chaos-FGD/blob/master/fgd/brush/func/func_slide.fgd#L6=